src/components/IKUpload/index.tsx

```diff
@@ -81,29 +96,49 @@ class IKUpload extends ImageKitComponent<IKUploadProps> {
       return;
     }
 
-    var ikClient = this.getIKClient();
+    var ikClient = getIKClient();
 
     const file = e.target.files?.[0];
     if (!file) {
       return;
     }
 
-    if (this.props.validateFile && !this.props.validateFile(file)) {
+    if (props.validateFile && !props.validateFile(file)) {
       return;
     }
-    if (this.props.onUploadStart && typeof this.props.onUploadStart === "function") {
-      this.props.onUploadStart(e);
+
+    if (props.onUploadStart && typeof props.onUploadStart === "function") {
+      props.onUploadStart(e);
     }
 
     const xhr = new XMLHttpRequest();
     const progressCb = (e: ProgressEvent<XMLHttpRequestEventTarget>) => {
-      if (this.props.onUploadProgress && typeof this.props.onUploadProgress === 'function') {
-        this.props.onUploadProgress(e);
+      if (props.onUploadProgress && typeof props.onUploadProgress === 'function') {
+        props.onUploadProgress(e);
       }
     };
 
     xhr.upload.addEventListener('progress', progressCb);
 
+    const allowedImageTypes = [
```

Comment:
Remove all this? We don't want SDK to do file type validation. We do that in backend.

Reply:
Removed